npm/pkg/dataplane/dataplane_windows.go

```diff
@@ -192,16 +192,16 @@ func (dp *DataPlane) updatePod(pod *updateNPMPod) error {
 			return err
 		}
 
-		endpoint.NetPolReference[policyName] = struct{}{}
+		endpoint.NetPolReference[policyKey] = struct{}{}
 	}
 
 	return nil
 }
 
-func (dp *DataPlane) getSelectorIPsByPolicyName(policyName string) (map[string]struct{}, error) {
-	policy, ok := dp.policyMgr.GetPolicy(policyName)
+func (dp *DataPlane) getSelectorIPsByPolicyName(policyKey string) (map[string]struct{}, error) {
```

Comment:
<nit> probably update this func name to getSelectorIPsByPolicyKey

Reply:
let me fix the followup PR?